pkg/remote/kyma_synchronization_context.go

```diff
@@ -129,20 +130,26 @@ func (c *KymaSynchronizationContext) ensureRemoteNamespaceExists(ctx context.Con
 	return nil
 }
 
-func (c *KymaSynchronizationContext) CreateOrUpdateCRD(ctx context.Context, plural string) error {
-	crd := &v1extensions.CustomResourceDefinition{}
+func (c *KymaSynchronizationContext) CreateOrUpdateCRD(ctx context.Context, plural string,
```

Comment:
Don't introduce cache to this function, CreateOrUpdateCRD only get called when there is no CRD found, we don't need to cache it.

Reply:
This function creates or updates the CRD for the runtime, but the cache is only for the KCP